common4j/src/main/com/microsoft/identity/common/java/AuthenticationConstants.java

```diff
@@ -588,6 +588,11 @@ public static final class OAuth2Scopes {
          */
         public static final String EMAIL_SCOPE = "email";
 
+        /**
+         * Scope to get transfer token
+         */
+        public static final String TRANSFER_TOKEN_SCOPE = "service::http://Passport.NET/purpose::PURPOSE_TRANSFER_TOKEN_BACKUP_RESTORE";
```

Comment:
If this is only ever going to be used in broker, can this be moved in broker4j/broker?